civihr_default_theme/templates/page/page--myleaves.tpl.php

```diff
@@ -0,0 +1,113 @@
+<?php
```

Comment:
This is the template of the _entire_ page, but you should only provide the template of the page content